website/content/docs/agent-and-proxy/autoauth/index.mdx

```diff
@@ -105,6 +105,10 @@ The top level `auto_auth` block has two configuration entries:
 
 - `sinks` `(array of objects: optional)` - Configuration for the sinks
 
+- `enable_reauth_on_new_credentials` `(bool: optional)` - If enabled, Auto-auth will
```

Comment:
In the code, this is documented as the following:
```
	// NOTE: This is unsupported outside of testing and may disappear at any
	// time.
```

As such, I don't feel as though it should be documented. I'm open to hearing if you feel otherwise, though. If we did want to support this and have a strong reason to do so, that would be a different conversation.

Reply:
I continued this question in the docs for `reload`.